odoo/fields.py

```diff
@@ -204,6 +204,7 @@ def _read_group_many2one_field(self, records, domain, order):
     column_type = None                  # database column type (ident, spec)
     column_format = '%s'                # placeholder for value in queries
     column_cast_from = ()               # column types that may be cast to this
+    write_order = 0                     # field ordering for write()
```

Comment:
nit: how about `write_priority` or `write_sequence`?

Obviously I'm no native speaker, but subjectively it feels like you assign a priority or a sequence number to a *specific* item, whereas an order is something you apply to a *collection* of items. Here I think we mean the former.

Reply:
You are right, the idea is to define 3 groups of fields; items within each group can be in any order.
X2many fields should be assigned last, monetary fields before last, and everything else first.
The name `write_sequence` looks good to me, as `write_priority` is confusing (lower value meaning higher priority).